data-prepper-plugins/opensearch/README.md

```diff
@@ -124,6 +124,8 @@ Default value is false. Set it to true for [Raw span trace analytics](#raw_span_
 Default value is false. Set it to true for [Service map trace analytics](#service_map_trace_analytics). Set it to false for [Raw span trace analytics](#raw_span_trace_analytics).
 
 - <a name="index"></a>`index`: A String used as index name for custom data type. Applicable and required only If index_type is explicitly `custom` or defaults to be `custom` while both `trace_analytics_raw` and `trace_analytics_service_map` are set to false.
+  * This index name can be a plain string, such as "application", "my-index-name".
+  * This index name can also be a plain string plus a date-time pattern as a suffix, such as "application-%{yyyy.MM.dd}", "my-index-name-%{yyyy.MM.dd.HH}". When OpenSearch Sink is sending data to OpenSearch, the date-time pattern will be replaced by actual UTC time. The pattern support all the symbols that represent one hour or above and are listed in [Java DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html). For example, with an index pattern like "my-index-name-%{yyyy.MM.dd}", a new index is created for each day such as "my-index-name-2022.01.25". 
```

Comment:
It might be useful to provide an example which includes time in the pattern i.e. "my-index-name-%{yyyy.MM.dd.HH}"

Nit: 

> The pattern supports all the symbols...

Reply:
👍  Sounds good. All addressed. Thanks.